File: sample-projects/go/application/dto/creature.go
package dto

type Creature struct {
	Id             string `json:"id"`
	Type           int    `json:"type"`
	HitPoints      int    `json:"hitPoints"`
	WalkingSpeed   int    `json:"walkingSpeed"`
	FlyingSpeed    int    `json:"flyingSpeed"`
	SwimmingSpeed  int    `json:"swimmingSpeed"`
	BurrowingSpeed int    `json:"burrowingSpeed"`
	ClimbingSpeed  int    `json:"climbingSpeed"`
	Stable         string `json:"stable"`
}

File: sample-projects/php/src/Application/Dto/Creature.php
<?php

declare(strict_types=1);

namespace De\Sots\CellarsAndCentaurs\Application\Dto;

final class Creature
{
    public function __construct(
        public readonly string $id,
        public readonly string $type,
        public readonly int $hitPoints,
        public readonly int $armorClass
    ) {
    }
}
